lib/compat/wordpress-6.5/fonts/class-wp-font-utils.php

```diff
@@ -20,11 +20,41 @@
 	 * @access private
 	 */
 	class WP_Font_Utils {
+
+		/**
+		 * Adds surrounding quotes to font family names that contain special characters.
+		 *
+		 * It follows the recommendations from the CSS Fonts Module Level 4.
+		 * @link https://www.w3.org/TR/css-fonts-4/#font-family-prop
+		 *
+		 * @since 6.5.0
+		 * @access private
```

Comment:
`@access private` probably isn't needed, since this is already a private method.

Reply:
Indeed!

Adding `@access private` is incorrect as per the coding standards.